common-libs/internal-api-clients/src/spec/_registry.ts

```diff
@@ -60,6 +60,9 @@ export default {
           },
           "publicKeyBase58": {
             "type": "string"
+          },
```

Comment:
I do not see this property in [CreateAnchorTransactionInput](https://gitlab.com/affinidi/foundational/affinity-registry/-/blob/main/src/did/did.dto.ts#L58), only in AnchorDidInput, which is already presented [here](https://github.com/affinityproject/affinidi-core-sdk/pull/232/files#diff-0874f74a245c4a669915ff56a14e08b4168ee64eaf80d50203ccf73a1af2c055R138).